Updating sharing options (#15913)
home-assistant/home-assistant.github.io: source/_docs/blueprint/tutorial.markdown

## Share the love

> The final step is to share this blueprint with others. 

Reviewer: ```suggestion
The final step is to share this blueprint with others.
```

### Informal Sharing

> For this tutorial we're going to share it on GitHub Gists. This is a good option if you don't want to publish your blueprint to a larger audience.

Reviewer: ```suggestion
For this tutorial, we're going to share it on GitHub Gists. This is a good option if you don't want to publish your blueprint to a larger audience.
```

- Click the "Create Gist" button

You can now copy the URL of your new Gist and share it with other people. They can import it by going to **Configuration**, **Blueprints** and clicking on the blue "Import Blueprint" button.

### Share on the Blueprint Exchange

> If you follow the [Rules and format for posting](https://community.home-assistant.io/t/about-blueprints/253788), you can share your blueprint on the Home Assistant Blueprint Exchange forum. This option is accessible to the general Home Assistant community, but recommended only for your original blueprints. Please don't post this tutorial to the Blueprint Exchange, but instead remember this as an option for releasing your real blueprints.

Reviewer: ```suggestion
If you follow the [Rules and format for posting](/get-blueprints), you can share your blueprint on the Home Assistant Blueprint Exchange forum. This option is accessible to the general Home Assistant community but recommended only for your original blueprints. Please don't post this tutorial to the Blueprint Exchange, but instead, remember this as an option for releasing your real blueprints.
```
